features/networking/sdn.feature

```diff
@@ -69,27 +69,18 @@ Feature: SDN related networking scenarios
     Given I select a random node's host
     And evaluation of `node.name` is stored in the :node_name clipboard
 
-    #Capturing the node's corresponding sdn pod to execute ovs commands on it
-    When I run the :get admin command with:
-       | resource      | pods                              |
-       | fieldSelector | spec.nodeName=<%= cb.node_name %> |
-       | namespace     | openshift-sdn                     |
-       | l             | app=sdn                           |
-       | output        | json                              |
-    Then the step should succeed
-    And evaluation of `@result[:parsed]['items'][0]['metadata']['name']` is stored in the :sdn_pod clipboard
- 
     #Below step will save plugin type and version value in net_plugin variable
     Given the cluster network plugin type and version and stored in the clipboard 
     #Changing plugin version to some arbitary value ff
     When I run command on the "<%= cb.node_name %>" node's sdn pod: 
       | ovs-ofctl| -O | openflow13 | mod-flows | br0 | table=253, actions=note:<%= cb.net_plugin[:type] %>.ff |
     Then the step should succeed
+    And evaluation of `pod` is stored in the clipboard
```

Comment:
This is not needed. Step `I run command on the "<%= cb.node_name %>" node's sdn pod:` already caches that pod. And in fact `pod` does not refer to the clipboard. `cb.pod` is what above step creates. So the line can just be removed.